third_party/libpg_query/grammar/statements/select.y

```diff
@@ -3735,6 +3747,11 @@ case_arg:	a_expr									{ $$ = $1; }
 			| /*EMPTY*/								{ $$ = NULL; }
 		;
 
+columnrefList:
+			columnref								{ $$ = list_make1($1); }
```

Comment:
`columnref` includes an optional indirection, meaning we can now also have `(tbl.col)`. Is that intentional? If it's not intentional, can we switch this to a `ColId`? If it is intentional, can we add a test for this?

Reply:
It's not intentional, I added a test with `col[1]` which does properly error, but perhaps I should just limit this at the parser level.
I was hesitant to because it would just mean duplicating existing rules in the grammar with a slight adaptation